scripts/popup.js

```diff
@@ -393,23 +393,41 @@ function show_wikibooks() {
 function noContextTip() {
   chrome.storage.sync.get(['alexa', 'domaintools', 'tweets', 'wbmsummary', 'annotations', 'tagcloud'], function (event) {
     // If none of the context is selected, grey out the button and adding tip when the user hovers
-    for (const context in event) { if (event[context]) { return $('#context-screen').click(show_all_screens) } }
-    if (!$('#ctxbox').hasClass('flip-inside')) { $('#ctxbox').addClass('flip-inside') }
+    for (const context in event) { 
+      if (event[context]) { 
+        $('#contextBtn').removeAttr('disabled')
+        return $('#contextBtn').click(show_all_screens) 
+      }
+    }
+    if (!$('#ctxbox').hasClass('flip-inside')) {
+      $('#ctxbox').addClass('flip-inside')
+      $('#contextBtn').attr('disabled', true)
```

Comment:
Is this line is required ? Please tell me a situation where it is useful

Reply:
This line is required if the user initially has no contexts selected, visits a non excluded URL and clicks on the extension icon. This line dims the 'Show Contexts' button in this case, same as the Back button. The dimmed button tells the user that no context is selected and the flip tells to select contexts in the settings.

Reply:
This make sense actually. Thank you for explaining !

Reply:
Thanks :-)